griptape/utils/observability/griptape_cloud_span_exporter.py

```diff
@@ -0,0 +1,49 @@
+import os
+from collections.abc import Sequence
+from urllib.parse import urljoin
+from typing_extensions import override
```

Comment:
We don't use `typing_extensions` anywhere else in the framework, I think this may be provided from a dependency.

Reply:
Removed, though it is nice to have an `@override` method to catch stupid naming typos that result in creating a new method instead of overriding an existing method.